### DISCUSSION TITLE
[Feature]: hide seekbar on native controls

### DISCUSSION THREAD

**Discussion by mezalejandro** [TheWidlarzGroup/react-native-video#4968] [Ideas]
### What problem are you trying to solve?

When playing a live broadcast using the native controls, the time/seek bar is not relevant and should not be shown. I want to hide the time bar in this case.

### Proposed solution

Provide a way, when using the native controls, to hide the time/seek bar (for example for live broadcasts).

### Platforms

iOS, Android

### Version line

v6

### Additional context

Architecture: Old architecture

Reproduction: repository link

Reproduction: Step to reproduce this bug are:

**Comment by freeboub** [comment]
No, I don't this is feature is feasible without code patch...

**Comment by seyedmostafahasani** [comment]
@mezalejandro this feature is available for Android on the latest version. You can use the [controlsStyles](https://thewidlarzgroup.github.io/react-native-video/component/props#controlsstyles) prop. I will try to add it on iOS.

**Comment by TWG-AI** [comment]
Thanks for the idea — noted for the team to consider.

> I'm an AI assistant, so double-check any specifics I may have mentioned.
